staging/src/k8s.io/apiserver/pkg/authentication/token/cache/cached_token_authenticator.go

```diff
@@ -86,7 +86,7 @@ func newWithClock(authenticator authenticator.Token, cacheErrs bool, successTTL,
 		// used. Currently we advertise support 5k nodes and 10k
 		// namespaces; a 32k entry cache is therefore a 2x safety
 		// margin.
-		cache: newStripedCache(32, fnvHashFunc, func() cache { return newSimpleCache(1024, clock) }),
+		cache: newStripedCache(32, fnvHashFunc, func() cache { return newSimpleCache(clock) }),
```

Comment:
Is there any reason to stripe any more?

Reply:
Even if it still makes sense, I bet we can go down to like 4 or 8 instead of 32?

Reply:
Striping still helps. Do you have strong feelings about fudging the number of shards in this change? If not, i'd just leave it.

Reply:
It's not important. If you wanted to make a change I would just try to ensure that the shards don't all GC at the same time, but even that is probably best for a followup.